nss-evening-cohort-14/gitsub-team-no-name
# Jm overview fontawesome GitHub

## Description
-Adding icons for bio information and highlights.
-Adding repository forks and stars with icons.
-Adding horizontal lines.
-Adding responsiveness

## Related Issue
https://github.com/nss-evening-cohort-14/gitsub-team-no-name/issues/23
( https://github.com/nss-evening-cohort-14/gitsub-team-no-name/issues/26 )

## Motivation and Context
Adds features found in GitHub, makes the page more interesting.

## How Can This Be Tested?
Load the Overview page,  Check for icons as well as horizontal lines in the bio sidebar and above the footer.

## Notes:
(About area, adding and removing pins for stretch goal already present from first pull request - issue 26. About area is modest. )

## Screenshots (if appropriate):
![Overview](https://user-images.githubusercontent.com/51683901/107897023-a6412300-6efd-11eb-8c69-c488b573a29e.png)

## Types of changes
- [x ] New feature (non-breaking change which adds functionality)
